.github/workflows/prettier.yml

```diff
@@ -7,12 +6,10 @@ on:
 jobs:
   prettier:
     runs-on: ubuntu-latest
-
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Check code
-        uses: creyD/prettier_action@31355f8eef017f8aeba2e0bc09d8502b13dbbad1
+      - uses: actions/checkout@v3
+      - name: sep-check
+        uses: DerYeger/yarn-setup-action@18fb87f9a712d8cc8ad1e7407a06470b397b4a7d
```

Comment:
This action appears to add little functionality. I think there's some value in keeping the dependencies light, especially ones that don't add much value. We could use the GitHub maintained `actions/setup-node` that also automatically installs yarn, and then we just need to have two `-run` steps, for `yarn install --frozen-lockfile` and `yarn sep-check`.

Reply:
SG, I will add a follow-up PR. I'm not familiar with node ecosystem tbh so I just added the first one I found

Reply:
https://github.com/stellar/stellar-protocol/pull/1519